docs/docs/recipes.md

```diff
@@ -853,6 +853,97 @@ export const pageQuery = graphql`
 
+### Sourcing from Wordpress
+
+#### Prerequisites
+
+- An existing [Gatsby site](/docs/quick-start/) with a `gatsby-config.js` and `gatsby-node.js` file
+- An existing template component such as `src/templates/post.js`
+
+#### Directions
+
+1. Install the `gatsby-source-wordpress` plugin by running the following command:
+
+```shell
+npm install gatsby-source-wordpress
+```
+
+2. Configure the plugin by modifying the `gatsby-config.js` file such that it includes the following:
+
+```JS:title=gatsby-config.js
+module.exports = {
+  ...
+  plugins: [
+    {
+      resolve: `gatsby-source-wordpress`,
+      options: {
+        // baseUrl will need to be updated with your wordpress source
+        baseUrl: `wpexample.com`,
+        protocol: `https`,
+        // is it hosted on wordpress.com, or self-hosted?
+        hostingWPCOM: false,
+        // does your site use the Advanced Custom Fields Plugin?
+        useACF: false
+      }
+    },
+  ]
+}
+```
+
+> > **Note:** Refer to the [`gatsby-source-wordpress` plugin docs](/packages/gatsby-source-wordpress/?=wordpre#how-to-use) to know more about configuring your plugins.
+
+3. Create dynamic pages for your Wordpress posts by pasting the following sample code in `gatsby-node.js`:
+
+```JS:title=gatsby-node.js
+const path = require(`path`)
+const slash = require(`slash`)
+
+exports.createPages = async ({ graphql, actions }) => {
+  const { createPage } = actions
+
+  // query content for WordPress posts
+  const result = await graphql(`
+    query {
+      allWordpressPost {
+        edges {
+          node {
+            id
+            slug
+          }
+        }
+      }
+    }
+  `)
+
+  const postTemplate = path.resolve(`./src/templates/post.js`)
+  result.data.allWordpressPost.edges.forEach(edge => {
+    createPage({
+      // `path` will be the url for the page
+      path: edge.node.slug,
+      // specify the component template of your choice
+      component: slash(postTemplate),
+      // In the ^template's GraphQL query, 'id' will be available
+      // as a GraphQL variable to query for this posts's data.
+      context: {
+        id: edge.node.id,
+      },
+    })
+  })
+}
+```
+
+4. Open the `GraphiQL IDE` at `localhost:8000/__graphql` and open the Docs or Explorer to observe the queryable fields for `allWordpressPosts`.
+
+5. Run `gatsby-develop` to see the newly generated pages and navigate through them.
+
+> **Note:** The dynamic pages created above have unique paths for navigating to particular posts, a template component for the posts, and a sample GraphQL query to source your Wordpress post content.
+
+#### Aditional resources
+
+- [Getting Started with Wordpress and Gatsby](/blog/2019-04-26-how-to-build-a-blog-with-wordpress-and-gatsby-part-1/)
+- More on [Sourcing from Wordpress](/docs/sourcing-from-wordpress/)
+- [Live example on Sourcing from Wordpress](https://watch-learn.com/series/gatsbyjs-wordpress)
```

Comment:
The watch-learn example is already linked from the Sourcing docs page. Rather than linking to it again here, we should include a link to your new example in the repo. 
```suggestion
- [Live example on Sourcing from Wordpress](/examples/gatsby-sourcing-wordpress/)
```

Reply:
Ahh, yes! I will replace it by linking my example once it's approved and merged in the repo.